tollway/vehicle.py

```diff
@@ -59,11 +63,8 @@ def create_tollway(tollways: dict) -> tuple[str, str]:
     return state, name
 
 
-def create_message(vehicle: dict[str, str], tollway: tuple[str, str]) -> dict[str, str]:
-    state_tollway = {
-        "tollway_state": tollway[0],
-        "tollway_name": tollway[1],
-    }
-    vehicle.update(state_tollway)
+def create_message(vehicle: dict, tollway: tuple[str, str]) -> dict[str, str]:
+    vehicle["tollway_state"] = tollway[0]
```

Comment:
named tuples are nicer

Reply:
agreed. Will update here.

Reply:
I'm going to create an issue for this too. I'm going to move all of vehicle into named tuple, I think, so will be touching few more areas.

Reply:
if it merits, I am also like, keep moving on with the new stuff ...

Reply:
agreed. I'll start with the other issues first and think on this one more. Or maybe just keep the issue open for future resolution.

Reply:
I know it's more effort but I just love the readability/cleanliness named tuples offer :)

Reply:
Yep that's what I love about the issues, we can just table and not forget about stuff that's better to address in the future. Btw on clean code, several people have recommended: https://www.amazon.es/Tidy-First-Personal-Exercise-Empirical/dp/1098151240/